### ISSUE TITLE
can I use gsplat in windows?

### ISSUE THREAD

**Issue by Ira569** [nerfstudio-project/gsplat#181]

**Comment by SimonF24** [comment]
I was just able to get it working on Windows but required a couple unorthodox steps after installing nerfstudio:
1. I ran the "ns-train splatfacto --data data/nerfstudio/poster" command from a 64-bit build tools administrator command prompt (the one with the title "x64 Native Tools Command Prompt for VS 2022", run as administrator). This requires installing the Visual Studio build tools if you don't already have them. It also wouldn't build in the x86 developer command prompt, had to be the 64-bit one.
2. I was using a venv so the python311.lib file (I'm using Python 3.11, I couldn't install nerfstudio with 3.12 due to the open3d dependency, which doesn't support 3.12) wasn't found by default so I had to create a libs folder in my venv Scripts folder and add the python311.lib file to it. The full path was /path/to/venv/Scripts/libs/python311.lib. The build seems to expect you're building with the default Python and tries to find the python311.lib file relative to the python.exe as far as I can tell (maybe conda environments would also let the python311.lib file be found? not sure).

After building gsplat the first time I was able to run it in the x64 Native Tools Command Prompt without running as administrator so I'm not sure if running as administrator was actually necessary. I was also able to run gsplat after deleting the libs folder I created in my venv Scripts folder
